### PR TITLE
create angular pwa app with comingsoon page

### PR DESCRIPTION
**Pull request by oshri**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,12254 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (eladraz) [inline yarn.lock]: Why do you have an auto-generated file?!

File: tslint.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
+{
```
Comment (eladraz) [inline tslint.json]: Do we need the linter here? I think it should be stored in a general shared repository location for many typescript applications.

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "./dist/out-tsc",
+    "sourceMap": true,
+    "declaration": false,
+    "module": "es2015",
```
Comment (eladraz) [inline tsconfig.json]: Why not 2018? (Stupid question)

File: tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      "node_modules/@types",
+			"src/typings"
+    ],
+    "lib": [
+      "es2018",
+      "dom"
+    ],
+    "paths": {
+      "@ns-core/*": ["src/app/modules/core/*"],
+      "@ns-material/*": ["src/app/modules/material/*"],
```
Comment (eladraz) [inline tsconfig.json]: Do we have materials?

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
+root = true
```
Comment (eladraz) [inline .editorconfig]: Please remove this file from the repo

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+# misc
+/.sass-cache
+/connect.lock
+/coverage
+/libpeerconnection.log
+npm-debug.log
+yarn-error.log
+testem.log
+/typings
```
Comment (eladraz) [inline .gitignore]: Please add .editorconfig and *.lock files

File: .jest-test-results.json [hunk 7]
```diff
@@ -0,0 +1 @@
...60339726141,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":["Module: Core"],"failureMessages":[],"fullName":"Module: Core should load","location":null,"status":"passed","title":"should load"}],"endTime":1560339735270,"message":"","name":"/Users/oshrikdoshim/projects/nextsilicon.github.io/src/app/modules/core/core.module.spec.ts","startTime":1560339726122,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":["Module: App"],"failureMessages":[],"fullName":"Module: App should load","location":null,"status":"passed","title":"should load"}],"endTime":1560339735337,"message":"","name":"/Users/oshrikdoshim/projects/nextsilicon.github.io/src/app/app.module.spec.ts","startTime":1560339726163,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":["HomeComponent"],"failureMessages":[],"fullName":"HomeComponent should create the app","location":null,"status":"passed","title":"should create the app"}],"endTime":1560339735643,"message":"","name":"/Users/oshrikdoshim/projects/nextsilicon.github.io/src/app/modules/core/containers/home/home.component.spec.ts","startTime":1560339726144,"status":"passed","summary":""}],"wasInterrupted":false}
```
Comment (eladraz) [inline .jest-test-results.json]: Please remove this file

File: .storybook/NextSilicon.svg [hunk 8]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (eladraz) [inline .storybook/NextSilicon.svg]: Please remove ".storybook" folder

File: tsconfig.json (hunk 3 above)
Comment (oshri) [reply tsconfig.json]: For old browser support 😊

File: tsconfig.json (hunk 4 above)
Comment (oshri) [reply tsconfig.json]: Yes, Material is our foundation ui, that we build on top of it.

File: yarn.lock (hunk 1 above)
Comment (oshri) [reply yarn.lock]: We used yarn.lock for lock all npm modules. It’s give us control on versioning Plus when we run tests in ci, the circleCi install all dependencies from lock file. So yes we need that.

File: tslint.json (hunk 2 above)
Comment (oshri) [reply tslint.json]: You right, if we decide to build monorepo so we will have one linter file. But for now we have different t repos.

File: .storybook/NextSilicon.svg (hunk 8 above)
Comment (oshri) [reply .storybook/NextSilicon.svg]: You right I can do that :)

File: .gitignore (hunk 6 above)
Comment (oshri) [reply .gitignore]: We need the Editor confit so every developer that develop in that repo will have the same configuration.

File: .editorconfig (hunk 5 above)
Comment (oshri) [reply .editorconfig]: We need the Editor confit so every developer that develop in that repo will have the same configuration.

**Pull request closed without merging**